From 7071a84c138ddb69085dd7f75e4c1cbeff39e400 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sat, 9 Jan 2016 17:22:17 +0000
Subject: [PATCH] More of bug 733137 - Don't force no static builds on Win32

And also: https://github.com/Alexpux/MINGW-packages/issues/987

From reading recent discussions, I believe static initiailizers
are handled OK now on all MSVC versions and MinGW-w64 GCC never
had any problems. So there is no need to avoid static libraries
globally (though avoiding them globally was never appropriate!)
---
 configure.ac | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0e1654f..a3f0781 100644
--- a/configure.ac
+++ b/configure.ac
@@ -136,17 +136,6 @@ case $host in
     ;;
 esac
 
-if test "$os_win32" = "yes"; then
-  if test x$enable_static = xyes -o x$enable_static = x; then
-    AC_MSG_WARN([Disabling static library build, must build as DLL on Windows.])
-    enable_static=no
-  fi
-  if test x$enable_shared = xno; then
-    AC_MSG_WARN([Enabling shared library build, must build as DLL on Windows.])
-  fi
-  enable_shared=yes
-fi
-
 dnl Initialize libtool
 LT_PREREQ([2.2.6])
 LT_INIT([disable-static win32-dll])
-- 
2.6.3

